packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.css

```diff
@@ -24,67 +20,40 @@ ul {
   width: 100%;
 }
 
-li {
+::slotted(li) {
   display: inline-block;
-  line-height: 1.5;
-  margin: 0.4rem 0;
+  line-height: 1.5 !important;
+  margin: 0.4rem 0 !important;
```

Comment:
These are here because external styles were affecting `li` somehow.

Reply:
This is coming from Formation.

<details><summary>vets-website</summary>
<img width="399" alt="Screen Shot 2022-02-17 at 10 15 32" src="https://user-images.githubusercontent.com/36863582/154511897-bbd9dbf7-ce78-45e8-b986-7f78fc4f37f8.png">
</details>

<details><summary>Storybook</summary>
<img width="396" alt="Screen Shot 2022-02-17 at 10 20 24" src="https://user-images.githubusercontent.com/36863582/154512822-97ff9123-e91d-457c-b155-91d63a8a14ab.png">
</details>

Reply:
I'm not keen on using `!important` here. And I'm wondering why Formation styles are overriding component styles in the cascade, because ideally it would be the other way around.

Reply:
Slotted content is technically in the light DOM, so global styles will still affect it.

Reply:
The latest changes on `vets-website`:

<img width="396" alt="Screen Shot 2022-02-17 at 13 26 21" src="https://user-images.githubusercontent.com/36863582/154546647-b04863bc-d77d-4b68-b6f4-6e663bf56e0f.png">

Reply:
Perfect, thank you!

Reply:
> shouldn't component styles be loaded after global styles in the cascade, thereby overriding them (unless there is a specificity issue)?

It's a specificity thing, unfortunately. [This answer](https://answerbun.com/stack-overflow/slotted-css-selector-for-nested-children-in-shadowdom-slot/#w) has some good explanations, specifically (no pun intended) from the section on the workaround for styling the light DOM:

> **Note: `::slotted([Simple Selector])`** confirms to Specificity rules,
but (being simple) does not add weight to lightDOM skin selectors, so never gets higher Specificity.
You might need `!important` in some (rare) use cases.

And the reason we can only use simple selectors inside `::slotted()` is for performance reasons.